server/src/main/java/org/apache/druid/server/coordinator/duty/CoordinatorDuty.java

```diff
@@ -17,26 +17,25 @@
  * under the License.
  */
 
-package org.apache.druid.server.coordinator.helper;
+package org.apache.druid.server.coordinator.duty;
 
+import org.apache.druid.server.coordinator.DruidCoordinator;
 import org.apache.druid.server.coordinator.DruidCoordinatorRuntimeParams;
 
 import javax.annotation.Nullable;
 
 /**
  *
  */
-public interface DruidCoordinatorHelper
+public interface CoordinatorDuty
```

Comment:
I don't understand that change, as `Duty` and `Helper` is of the same abstract level for me.

Reply:
There are dozens of different "Helpers" in all parts of Druid. This is a meaningless term. "Duty" is a distinctive term, and it does carry some meaning - "a repetitive task that a coordinator has to do".

Reply:
I'm confused again. I just did a spot check and it gave me a bunch of different results which don't look like yours:
![image](https://user-images.githubusercontent.com/14215045/62380720-7700d600-b552-11e9-9c75-2744b59a4cc9.png)
If you're serious, I wouldn't like to argue on that level of abstraction anymore. I see this change as redundant, that's my professional opinion, you can ignore it, of course.

Reply:
That's translation 2) in your screenshot. `CoordinatorDuty` objects are literally "tasks or actions that Coordinator is required to perform as part of its job".